apps/showcase/src/app/app.component.html

```diff
@@ -1,11 +1,38 @@
-<div class="container-fluid d-flex gap-2 h-100">
-  <div class="left-nav d-flex flex-column">
-    <o3r-sidenav-pres
-      [linksGroups]="linksGroups"
-      [activeUrl]="activeUrl$ | async"
-    ></o3r-sidenav-pres>
+<header>
+  <div class="header-icon">
+    <img src="assets/otter-sticker.svg" width="48" height="48" alt="Otter logo" />
   </div>
-  <div class="page-content flex-fill">
-    <router-outlet></router-outlet>
+  Otter Showcase
+</header>
+<div class="container">
+  <div class="row">
+    <div class="left-nav col-2 d-none d-xl-block">
+      <o3r-sidenav-pres
+        [linksGroups]="linksGroups"
+        [activeUrl]="activeUrl$ | async"
+      ></o3r-sidenav-pres>
+    </div>
+    <div class="page-content col-12 col-xl-10">
+      <router-outlet></router-outlet>
+    </div>
   </div>
 </div>
+<footer>
+  <div class="container">
+    <div class="row">
+      <div class="col-8">
+      </div>
+      <div class="col-4">
+        <h5>Links</h5>
```

Comment:
missing loc

Reply:
do we really use the translation reco from otter in the showcase?

Reply:
If the "showcase" is a showcase of the Otter capabilities, I would say yes :)

Reply:
it needs to be done globally on all the texts of the showcase app, it should be done in a separate PR